data-imports/clean_utils.py

```diff
@@ -67,14 +67,35 @@ def make_index(table, column):
     """
     return "ALTER TABLE {} ADD INDEX {};".format(table, column)
 
+def column_exists(table, column, schema='deltanyc'):
+
+    sql = """
+        SELECT column_name FROM information_schema.columns WHERE
+            table_schema = '{schema}'
+            AND table_name = '{table}'
+            AND column_name = '{column}'
+        """.format(schema=schema, table=table, column=column)
+
+    result = connect().execute(sql)
+
+    return (result.rowcount == 1)
+
 def clean_bbl(table, boro, block, lot):
-    return '''
-    ALTER TABLE {table} ADD COLUMN bbl bigint(13) NULL DEFAULT NULL;
+    sql = '''
+    UPDATE {table} SET {block} = REPLACE({block}, ' ', 0);
+    UPDATE {table} SET {block} = REPLACE({block}, '`', '');
+    UPDATE {table} SET {lot} = REPLACE({lot}, ' ', 0);
```

Comment:
It's late for this comment, but: maybe best to just preg_replace all non-digit characters?